luminance/src/state.rs

```diff
@@ -116,6 +116,18 @@ impl GraphicsState {
     }
   }
 
+  /// Help swap from an opengl context to another
+  /// 
+  /// Note: currently it only run the same code as in new() method, but without the check
+  /// to the TLS to be sure that only one context exist in the thread
+  /// WARNING : you can use this method to bypass the new() unique context security, but
+  /// it's a really bad idea, please consider this method as an advanced feature and use it
+  /// with caution.
+  #[cfg(feature = "multiwin")]
+  pub fn new_multi() -> Result<Self, StateQueryError> {
```

Comment:
The name of the function is not clear enough. `new_unchecked` would be better to me.

Reply:
`new_unchecked` looks also unclear, as it's not unchecked, because you can't really check from luminance something only working on the window manager side. I changed for `new_multi_contexts`, seems better and align to the feature.